### PR TITLE
Create migration for the application

### PR DESCRIPTION
**Pull request by maiklgravi**

### CODE REVIEW

File: database/migrations/2021_12_11_002430_film_info_table.php [hunk 1]
```diff
@@ -17,12 +17,11 @@ public function up()
...
-            $table->string('screenwriter');
-            $table->string('actors');
-            $table->string('country');
+            $table->string('lang' , 15);
+            $table->string('screenwriter' , 25);
+            $table->string('actors' , 50);
+            $table->string('country' , 15);
             $table->integer('duration');
-            $table->timestamps();
             $table->foreign('id_film')->references('id')->on('film')->cascadeOnUpdate()->cascadeOnDelete();
```
Comment (maiklgravi) [inline database/migrations/2021_12_11_002430_film_info_table.php:25]: here i will fix

File: database/migrations/2021_12_11_001931_film_video_name_table.php [hunk 2]
```diff
@@ -18,7 +18,6 @@ public function up()
             $table->id();
             $table->integer('id_film');
             $table->string('name', 255);
-            $table->timestamps();
             $table->foreign('id_film')->references('id')->on('film')->cascadeOnUpdate()->cascadeOnDelete();
```
Comment (maiklgravi) [inline database/migrations/2021_12_11_001931_film_video_name_table.php:21]: here i will fix

File: database/migrations/2021_12_10_234843_film_alternative_name_table.php [hunk 3]
```diff
@@ -17,8 +17,7 @@ public function up()
         Schema::create('film_alternative_name', function (Blueprint $table) {
             $table->id();
             $table->integer('id_film');
-            $table->string('name', 255);
-            $table->timestamps();
+            $table->string('name', 25);
             $table->foreign('id_film')->references('id')->on('film')->cascadeOnUpdate()->cascadeOnDelete();
```
Comment (maiklgravi) [inline database/migrations/2021_12_10_234843_film_alternative_name_table.php]: here i will fix

File: app/Models/FilmInfo.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class FilmInfo extends Model
+{
+    use HasFactory;
+    protected $table='film_info';
+    protected $fillable = array('name', 'film_id','duration','actors','screenwriter');
```
Comment (vistrati) [inline app/Models/FilmInfo.php:12]: in your table you don't have column `name`, also check if all columns are present in fillable

File: app/Models/Payment.php [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Payment extends Model
+{
+    use HasFactory;
+    protected $fillable = array('id_user', 'month', 'date_payment','payment');
```
Comment (vistrati) [inline app/Models/Payment.php:11]: add `value` in fillable

**Pull request merged**